conf/application.conf

```diff
@@ -3,4 +3,19 @@ play.i18n.langs=["en"]
 play.http.secret.key="changeme"
 play.http.secret.key=${?APPLICATION_SECRET}
 play.modules.enabled += "uk.gov.nationalarchives.omega.editorial.modules.StartupModule"
-play.http.filters=uk.gov.nationalarchives.omega.editorial.support.filter.CustomFilters
\ No newline at end of file
+play.http.filters=uk.gov.nationalarchives.omega.editorial.support.filter.CustomFilters
+
+# JMS client takes appox 20 seconds to shut down - we increase the coordinated
```

Comment:
Good and useful comment.

However - there is a typo: "appox".

Reply:
Oof, thanks, fixed.

Reply:
Confirmed.